src/main/scala/com/databricks/labs/overwatch/pipeline/SilverTransforms.scala

```diff
@@ -757,10 +757,11 @@ trait SilverTransforms extends SparkSessionWrapper {
           'spark_conf,
           'driver_instance_pool_id,
           'instance_pool_id,
-          'spark_version,
+          'effective_spark_version.alias("spark_version"),
```

Comment:
all added fields need to be added to the appropriate minimumSchema in Schema.scala. Be sure this is in the requestParams section of audit_log_master_schema and that the downstream consumers of these tables have them -- if they are missing and placed in a select it will blow up the pipeline.

This is true for effective_spark_version and runtime_engine (only two I see added ATM.